studio/pages/new/index.tsx

```diff
@@ -10,6 +10,25 @@ import { WizardLayout } from 'components/layouts'
 import Panel from 'components/ui/Panel'
 import { NextPageWithLayout } from 'types'
 
+const ORG_KIND_TYPES = {
+  PERSONAL: 'Personal',
+  EDUCATIONAL: 'Educational',
+  STARTUP: 'Startup',
+  AGENCY: 'Agency',
+  COMPANY: 'Company',
+  UNDISCLOSED: 'Don’t know',
```

Comment:
Just a thought, but 'N/A' feels better to me here than 'Don’t know'. Not a strong opinion – I can be persuaded back if you think it's good how it is!

Reply:
@alaister changed!